### PR TITLE
Modified for feedback on setup section

### PR DESCRIPTION
**Pull request by ArchanaShukla**
…rom QA to Production

### CODE REVIEW

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md [hunk 1]
```diff
@@ -26,25 +26,31 @@ To create service instances for business process management services, you need t
 
 1. In your web browser, open the [SAP Cloud Platform Trial cockpit](https://account.hanatrial.ondemand.com/cockpit).
 
-2. Navigate into the trial subaccount
+2. Click on **Enter Your Trial Account** to navigate into the trial account
```
Comment (MichaelCzcz) [inline tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md]: We like to say "click" not "click on".

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md [hunk 2]
```diff
@@ -26,25 +26,31 @@ To create service instances for business process management services, you need t
...
 
       !![Open Trial Account](opentrialaccount.png)
 
     >The trial account should contain one `subaccount` and `space`
 
-3. Open **Entitlements** from left panel. You should have the following entitlements in your account.
+3. Click on the subaccount.
```
Comment (MichaelCzcz) [inline tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md]: We like to say "click" not "click on".

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md [hunk 3]
```diff
@@ -26,25 +26,31 @@ To create service instances for business process management services, you need t
...
-
-    - Portal and HTML5 Application is needed to access Process Visibility and Workflow applications from Fiori Launchpad
+    |  Entitlement     | Description
+    |  :------------- | :-------------
+    |  Business Rules, Workflow and Process Visibility | These are core services of Intelligent BPM which will be needed to run the next tutorials.
+    |  Portal, Application Runtime, HTML5 Applications | These services are needed to setup and access Process Visibility and Workflow applications from Fiori Launchpad
+    |  Connectivity    | This service is needed to create and configure destinations
 
-    - Business Rules, Workflow and Process Visibility is needed to create service instances of these respective services in order to work with them.
+    - If you do not see these entitlements in your subaccount, then do the following:
```
Comment (MichaelCzcz) [inline tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md]: No need for bullet on this line.

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md [hunk 4]
```diff
@@ -26,25 +26,31 @@ To create service instances for business process management services, you need t
...
+        - click **Configure Entitlements**
```
Comment (MichaelCzcz) [inline tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md]: For these four sentences please capitalize and finish with a period.
The bullets are OK.

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md [hunk 5]
```diff
@@ -26,25 +26,31 @@ To create service instances for business process management services, you need t
...
+        - next click on **Add Service Plans**
```
Comment (MichaelCzcz) [inline tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md]: We like to say "click" not "click on".

**Comment by MichaelCzcz** [comment]
> Yeah the changes are fine

The PR is still in status "Cheanges requested". Could you please appy the suggested changes? Thank you.

**Comment by ArchanaShukla** [comment]
Check now!

**Pull request merged**